### DISCUSSION TITLE
PHP runtime: problem with operation on a directory

### DISCUSSION THREAD

**Discussion by f3l1x** [vercel/vercel#9239] [General] [answered]
Hi ✋ 

We (php folks) are now facing this problem. With no change at vercel-php it starts failing.

```
[17:39:48.711] Cloning github.com/hatsuyuki2333/phimg (Branch: main, Commit: 2bf8ba9)
[17:39:49.024] Previous build cache not available
[17:39:56.013] Cloning completed: 7.301s
[17:39:56.206] Running "vercel build"
[17:39:56.833] Vercel CLI 28.11.1
[17:39:57.066] > Installing Builder: vercel-php@0.5.2
[17:40:02.141] 🐘 Downloading user files
[17:40:02.141] 🐘 Downloading PHP runtime files
[17:40:02.237] 🐘 Creating lambda
[17:40:03.142] Error: EISDIR: illegal operation on a directory, read
```

More details from different users are here: https://github.com/vercel-community/php/issues/355

I'd appreciate some help from runtime team. Thank you for.

**Comment by gs0510** [comment]
Hi @f3l1x! Thank you for reporting this. Would you be able to provide the deployment id for this failing build? The deployment id is in the url of the deployment, something like: `https://vercel.com/<vercel_user>/vercel-php/<deployment_id>`. Thank you!

**Reply by tao** [reply]
I tried to get it working with a new Laravel Breeze app today and it failed, you can try this deployment id? 9PjSS8BAjctCFbaf4mz3LHSknsCV

**Reply by tao** [reply]
This one is connected to a [git repository](https://github.com/tao/vercel-php-laravel-breeze) you can try deploy yourself: Cy7R54WaSZYoPFuAjQByjmN5AhEn

**Reply by thedeadofblackfire** [reply]
HI,
Here a deployment id that just failed : 79Gmfz9B3ALrDKvvCiJhsKVfHxZm
Thks

**Comment by tao** [comment]
All my PHP apps are not deploying anymore, I had to move some off Vercel already cause they wouldn't update.

**Comment by gs0510** [comment]
Thanks for all the deployment ids, I'm taking a look at the issue and will report back as soon as I have a diagnosis and/or workaround.

**Comment by leerob** [comment, marked as answer]
Hey there, we believe this should be fixed on the Vercel side. Please let us know! 🙏

**Reply by thedeadofblackfire** [reply]
Hi, Yes it works now on various projects using vercel-php. Thanks for the fast fix.
